src/app/modules/angular-slickgrid/editors/autoCompleteEditor.ts

```diff
@@ -69,7 +71,7 @@ export class AutoCompleteEditor implements Editor {
     const title = this.columnEditor && this.columnEditor.title || '';
     this.labelName = this.customStructure && this.customStructure.label || 'label';
     this.valueName = this.customStructure && this.customStructure.value || 'value';
-
+    this.forceUserInput = this.columnEditor && this.columnEditor.editorOptions && this.columnEditor.editorOptions.forceUserInput ? this.columnEditor.editorOptions.forceUserInput : false;
```

Comment:
To simplify this line, you could maybe use a Getter, for example see this [Getter](https://github.com/ghiscoding/Angular-Slickgrid/blob/c4a29417c01de8fdf81a999afa25a517a6c1e5ce/src/app/modules/angular-slickgrid/editors/autoCompleteEditor.ts#L50). It's just a suggestion, I could also accept the code without it too

Reply:
Done, makes sense to refactor it  :)

Reply:
Oh man I'm really sorry, I was not clear in my comment, can you change it to the following
```ts
get editorOptions() {
    return this.columnEditor && this.columnEditor.editorOptions || {};
  }
```
then you can call your boolean values (and any other editor options) like so
```ts
serializeValue() {
  if (this.editorOptions.forceUserInput) {
...
```

Reply:
No problem at all (never used getter before) made a new pr, let me know if it is what you have in mind. Thanks